setup_env.bat

```diff
@@ -24,6 +24,7 @@ call git submodule update
 
 call android.bat update project -p libs/android-support-appcompat-v7-exploded-aar --target android-23
 call android.bat update project -p libs/android-support-design-exploded-aar --target android-23
+call android.bat update project -p libs/com-getbase-floatingactionbutton-1-10-1-exploded-aar --target android-23
```

Comment:
DONE, see https://github.com/owncloud/android/commit/e4d936e83754b823c90160889ce4704dfc3bbe61 - kept v22 since it doesn't make a difference anyways since the lib is in binary form thus compiled against v22 anyways... :D